KorolenkoDA/lab_1/Dima1.cpp

```diff
@@ -0,0 +1,180 @@
+﻿
+#include <iostream>
+#include <vector>
+#include <stdio.h>
+#include <string>
+
+using namespace std;
+
+struct Word
+{
+    int numberInArray;
+    int sumASCII;
+};
+
+vector<string> arrayWords, arrayWithoutMarks;

```

Comment:
Please get rid of global variables which is usually a bad coding practice
Functions are designed for that purpose: you can return result from function instead of modifying global state and you can pass one function result to another